src/reduced_basis/rb_construction.C

```diff
@@ -702,15 +708,12 @@ void RBConstruction::add_scaled_matrix_and_vector(Number scalar,
           context.get_elem_jacobian() *= 0.5;
         }
 
-      if (apply_dof_constraints)
-        {
-          // Apply constraints, e.g. Dirichlet and periodic constraints
-          this->get_dof_map().constrain_element_matrix_and_vector
-            (context.get_elem_jacobian(),
-             context.get_elem_residual(),
-             context.get_dof_indices(),
-             /*asymmetric_constraint_rows*/ false );
-        }
+      // Apply constraints, e.g. Dirichlet and periodic constraints
```

Comment:
This comment should mention that Dirichlet constraints have been removed, if that's what the stashing has done, right?